tdp/cli/commands/conftest.py

```diff
@@ -1,11 +1,35 @@
 # Copyright 2022 TOSIT.IO
 # SPDX-License-Identifier: Apache-2.0
 
+from collections.abc import Generator
 from pathlib import Path
 
 import pytest
+from click.testing import CliRunner, Result
+from sqlalchemy import create_engine
 
+from tdp.cli.commands.init import init
 from tdp.conftest import generate_collection_at_path
+from tdp.core.models import BaseModel
+
+
+@pytest.fixture
+def tdp_init(
+    collection_path: Path, db_dsn: str, vars: Path
+) -> Generator[Result, None, None]:
+    args = [
+        "--collection-path",
+        collection_path,
+        "--database-dsn",
+        db_dsn,
+        "--vars",
+        vars,
+    ]
+    runner = CliRunner()
+    yield runner.invoke(init, args)
+    engine = create_engine(db_dsn)
+    BaseModel.metadata.drop_all(engine)
+    engine.dispose()
```

Comment:
This should run `tdp init` and then yield `collection-path`, `database-dsn` and `vars` (as a named tuple or directly as an usable table like `args` in the current fixture).

Reply:
Has been changed, a bit complicated with the typing of the fixture.